content/docs/command-reference/init.md

```diff
@@ -14,106 +14,103 @@ DVC works best in a Git repository. This enables all features, providing the
 
-- Not enough isolation/granularity - DVC config, cache, and other files are
-  shared across different sub-projects. Means that it's not easy to use
-  different remote storages, for example, for different sub-projects, etc.
-
-- Not enough isolation/granularity - commands like `dvc pull`, `dvc checkout`,
-  and others analyze the whole repository to look for `dvc.yaml` or `.dvc` files
-  to download files and directories, to reproduce <abbr>pipelines</abbr>, etc.
-  It can be expensive in the large repositories with a lot of projects.
-
-- Not enough isolation/granularity - commands like `dvc metrics diff`, `dvc dag`
-  and others by default dump all the metrics, all the pipelines, etc.
+- By default, DVC commands like `dvc checkout` and `dvc repro` explore the whole
+  <abbr>DVC repository</abbr> to find DVC-tracked data and pipelines to work
+  with. This can be undesirable and inefficient for large monorepos.
 
 #### How does it affect DVC commands?
 
-No matter what mode is used, DVC looks for the `.dvc` directory when it starts
-(from the current working directory and up). Location of the found `.dvc`
-directory determines the root of the DVC project. (In case of `--subdir` it
-might happen that Git repository root is located at different path than the DVC
-project root.)
+The <abbr>project</abbr> root is found by DVC by looking for `.dvc/` from the
+current working directory, up. It defines the scope of action for most DVC
+commands (e.g. `dvc repro`, `dvc pull`, `dvc metrics diff`), meaning that only
+`dvc.yaml`, `.dvc` files, etc. inside the project are usable by the commands.
 
-DVC project root defines the scope for most DVC commands. Mostly meaning that
-all `dvc.yaml` and `.dvc` files under the root path are being analyzed.
+With `--subdir`, the project root will be found before the Git root, making sure
+the scope of DVC commands run here is constrained to this project alone, even if
+there are more DVC-related files elsewhere in the repo. Similarly, DVC commands
+run outside this project root will ignore its contents.
 
-If there are multiple DVC sub-projects but they _are not_ nested, e.g.:
+If there are multiple `--subdir` projects, but not nested, e.g.:
 
-```
-.
+```dvc
+.           # plain Git repo
 ├── .git
-|
 ├── project-A
-│   └── .dvc
+│   ├── .dvc    # dvc init --subdir
 │   ...
 ├── project-B
-│   └── .dvc
+│   ├── .dvc    # dvc init --subdir
 │   ...
 ```
 
-DVC considers them a two separate DVC projects. Any DVC command that is being
-run in the `project-A` is not aware about DVC `project-B`. DVC does not consider
-Git repository root an initialized DVC project in this case and commands that
-require DVC project will raise an error.
-
-On the other hand, if there _are_ nested DVC projects, e.g.:
+DVC considers A and B separate projects. Any DVC command run in `project-A` is
+not aware of `project-B`. However, commands that involve versioning (like
+`dvc checkout`) access the commit history from the Git root (`.`).
```

Comment:
`dvc checkout` does not require GIt

Reply:
True, it's not the best example. Changed to `diff` now.